packages/smooth_ui_library/lib/dialogs/smooth_AlertDialog.dart

```diff
@@ -0,0 +1,189 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_ui_library/buttons/smooth_simple_button.dart';
+
+
+///
+///	Open by calling 
+///
+///showDialog<void>(
+///        context: context,
+///        builder: (BuildContext context) {
+///          return SmoothAlertDialog(...)
+///	}
+///)
+///
+
+class SmoothAlertDialog extends StatefulWidget {
+  final BuildContext context;
+  final String title;
+  final bool close;
+  final double height;
+  final Widget body;
+  final List<SmoothSimpleButton> actions;
+
+  SmoothAlertDialog(
+      {Key key,
+        this.context,
+        this.title,
+        this.close  = true,
+        this.height,
+        this.body,
+        this.actions
+      })
+      : super(key: key);
+
+  @override
+  _SmoothAlertDialogState createState() => _SmoothAlertDialogState(
+    key: key,
+    context: context,
+    title: title,
+    close: close,
+    height: height,
+    body: body,
+    actions: actions,
+  );
+}
+
+class _SmoothAlertDialogState extends State<SmoothAlertDialog> {
+  final BuildContext context;
+  final String title;
+  final bool close;
+  final double height;
+  final Widget body;
+  final List<SmoothSimpleButton> actions;
+
+  _SmoothAlertDialogState({Key key,
+    this.context,
+    this.title,
+    this.close,
+    this.height,
+    this.body,
+    this.actions});
+
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(20.0)), //this right here
+
+      content: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            _buildTitle(title),
+            Container(
+              height: height,
+              child: body,
+            ),
+          ]
+      ),
+
+
+      actions: [
+        SizedBox(
+          height: 58,
+          width: MediaQuery
+              .of(context)
+              .size
+              .width,
+          child: Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            // Credits to https://stackoverflow.com/a/64697189/13313941
+            children: actions,
+          ),
+        ),
+      ],
+    );
+  }
+
+
+  Widget _buildTitle(String _title) {
+    double height = 29;
+
+    if (_title == null) {
+      return Container();
+    }
+    else {
+      return Column(
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+
+              _buildCrossPlaceHolder(),
```

Comment:
Should be possible to replace the _buildCrossPlaceHolder() function by a simple `SizedBox(width: close ? height : null),` :slightly_smiling_face:

Reply:
I made a conscious decision to do this in a function so that either both or neither is built. So that the title is always in the center, However, I have revised the function. Now there is only one for the placeholder and one for the button itself.